bindings/cs/MongoDB.Libmongocrypt/SigningRSAESPKCSCallback.cs

```diff
@@ -0,0 +1,75 @@
+﻿/*
+ * Copyright 2020–present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Runtime.InteropServices;
+#if NETSTANDARD2_1
+using System.Security.Cryptography;
+#endif
+
+namespace MongoDB.Libmongocrypt
+{
+    internal static class SigningRSAESPKCSCallback
+    {
+#pragma warning disable IDE1006 // Naming Styles
+        public static bool rsaSign(
```

Comment:
Feel free to change the name of the method to avoid the warning.

Reply:
done